tests_proc_dict.py

```diff
@@ -65,35 +81,97 @@ def drop_annotation(continuous_annotation):
 @pytest.fixture
 def user_dict():
     return {
-        "age": {
-            "Description": "age of the participant",
-            "Units": "years"
-        },
+        "age": {"Description": "age of the participant", "Units": "years"},
         "sex": {
             "Description": "gender of the participant",
             "Levels": {"M": "male", "F": "female"},
         },
     }
 
 
-def test_original_data_unchanged_when_no_annotation(drop_annotation, user_dict):
+def test_original_data_unchanged_when_no_annotation(
+    drop_annotation, user_dict
+):
     """Dropped annotations should not be added to the generated data dictionary"""
     data = pd.DataFrame(drop_annotation)
     result = process_dict(data, user_dict)
     assert result == user_dict
 
 
-def test_original_data_augmented_by_annotation(continuous_annotation, user_dict):
+def test_original_data_augmented_by_annotation(
+    continuous_annotation, user_dict
+):
     data = pd.DataFrame(continuous_annotation)
     result = process_dict(data, user_dict)
-    
+
     assert result.get("age", {}).get("Annotations") is not None
     assert result.get("sex") is not None
-    
-    
+
+
+def test_good_continuous_has_transformation(continuous_annotation, user_dict):
+    data = pd.DataFrame(continuous_annotation)
+    result = process_dict(data, user_dict)
+
+    assert (
+        result.get("age").get("Annotations").get("Transformation") is not None
+    )
+
+
+def test_bad_continuous_lacks_transformation(continuous_annotation, user_dict):
+    continuous_annotation.update(**{"type": {10: "nonsense_heuristic"}})
+    data = pd.DataFrame(
+        continuous_annotation
+    )
+    result = process_dict(data, user_dict)
+
+    assert result.get("age").get("Annotations", {}).get("Transformation") is None
```

Comment:
🍒 : Hope I'm not hallucinating, but I think we only really care about the `"Annotations"` in this scenario (and that it wasn't added, since no `"Transformation"` was added),
```suggestion
    assert result.get("age").get("Annotations") is None
```